addons/account_payment/models/account_move.py

```diff
@@ -39,8 +40,12 @@ def _compute_amount_paid(self):
             )
 
     def _has_to_be_paid(self):
+        # FIXME ANVFE do we only wanna consider last tx or all txs?
+        # In portal template we consider last tx sometimes, all txs sometimes....
```

Comment:
All. Let's be consistent with the model-object, especially if it was somewhat supported before.

Reply:
We should drop this `_get_portal_last_tx` logic sometimes then